sunpy/net/dataretriever/sources/tests/test_goes_ud.py

```diff
@@ -23,27 +23,29 @@ def LCClient():
 @pytest.mark.remote_data
 @pytest.mark.parametrize(
     "timerange,url_start,url_end",
-    [(TimeRange('1995/06/03', '1995/06/05'),
+    [(Time('1995/06/03', '1995/06/05'),
```

Comment:
just a general comment - we should probably have some tests that also include %H%m to check this works too

Reply:
Added many tests with `%H%M` for goes.